site/_docs/reference.md

```diff
@@ -2729,6 +2729,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | o | EXTRACT(xml, xpath, [, namespaces ])           | Returns the XML fragment of the element or elements matched by the XPath expression. The optional namespace value that specifies a default mapping or namespace mapping for prefixes, which is used when evaluating the XPath expression
 | o | EXISTSNODE(xml, xpath, [, namespaces ])        | Determines whether traversal of a XML document using a specified xpath results in any nodes. Returns 0 if no nodes remain after applying the XPath traversal on the document fragment of the element or elements matched by the XPath expression. Returns 1 if any nodes remain. The optional namespace value that specifies a default mapping or namespace mapping for prefixes, which is used when evaluating the XPath expression.
 | m | EXTRACTVALUE(xml, xpathExpr))                  | Returns the text of the first text node which is a child of the element or elements matched by the XPath expression.
+| h s | FIND_IN_SET(string, stringArray)             | Returns the index (1-based) of the given *string* in the comma-delimited *stringArray*. Returns 0, if the given *string* was not found or if *string* contains a comma
```

Comment:
From this description I infer that stringArray is of type `ARRAY STRING`, however it seems to be just a string which may contain commas. (I also think that the function name is not a very good choice, but that cannot be changed.)

Reply:
Good advice, the parameter name can be improved to a more concise one, like `matchStr` and `textStr`, to avoid ambiguity.

Reply:
I think we can add an example to clarify it, like `FIND_IN_SET('bc', 'a,bc,def')` returns TRUE.

Reply:
Updated.